packages/temporal/workflows/mtv_test.go

```diff
@@ -382,32 +362,33 @@ func (s *UnitTestSuite) Test_JoinCreatedRoom() {
 		s.Len(mtvState.Users, 2)
 	}, 3*time.Second)
 
-	s.env.ExecuteWorkflow(workflows.MtvRoomWorkflow, params)
+	s.env.ExecuteWorkflow(MtvRoomWorkflow, params)
 
 	s.True(s.env.IsWorkflowCompleted())
 	err := s.env.GetWorkflowError()
 	s.ErrorIs(err, workflow.ErrDeadlineExceeded, "The workflow ran on an infinite loop")
 }
 
 // Test_GoToNextTrack scenario:
-//
+
```

Comment:
We should keep the empty comments. If you hover the function `Test_GoToNextTrack` you will see that its description only begins from the fifth point. Empty comments are intended to allow line breaks.

Reply:
Done !